File: java eclipse/company/src/group/target1.java
package group;

import java.util.Scanner;

public class target1 {

	public static void main(String[] args) {
		
		int target , n ,x = 0 ;
		System.out.println("How many numbers you want to entered");
		Scanner sc = new Scanner(System.in);
		n = sc.nextInt();		
		int arr[] = new int[n];
		

		for(int a = 0 ;a<n;a++) {
			System.out.println("Enter number ");
			arr[a]=sc.nextInt();
		}
		
		for(int i = 0 ;i<n;i++) {
			System.out.print(arr[i]+ ",");
		}
		
		System.out.println("\nEnter target ");
		target = sc.nextInt();
		
		try {
		
		for(int i = 0 ;i<n;i++) {
			
			x= arr[i]+arr[i+1];
			if(x==target) 
				System.out.println(arr[i]+" "+ arr[i+1]);
		}
		
		if(x!=target) {
			System.out.println("Target is Not Exist ");
		}
		
		}
		catch(Exception e) {
			System.out.println("We known last index is array out of bound ");
		}
		
		
	}

}

File: c++mca/group/target1.cpp
#include<iostream>
using namespace std ;

int main(){
	int target , n;
	
	cout<<"How many numbers you have to entered in array ";
	cin>>n;
	
	int arr[n];
	
	for(int a=0; a<n;a++)
		cin>>arr[a];
	
	
	for(int i=0;i<n;i++)
		cout<<arr[i]<<" ";
		
    
    cout<<"\nEnter target ";
    cin>>target;
    
    for(int i=0;i<n;i++){
        
        int k = arr[i];
        k = k + arr[i+1];
        if(k==target){
            cout<<"["<<arr[i]<<","<<arr[i+1]<<"]";
            break;
        }
    }
    
    
    
	
	
	
	
	return 0 ;
}